cache-config/t3c-apply/config/config.go

```diff
@@ -188,6 +190,29 @@ func directoryExists(dir string) (bool, os.FileInfo) {
 	return info.IsDir(), info
 }
 
+const RpmDir = "/var/lib/rpm"
+
+// verifies the rpm database files. if there is any database corruption
+// it will return false
+func VerifyRpmDB() bool {
```

Comment:
GoDoc comments on exported symbols should begin with the name of the symbol they document (optionally following an article), contain complete sentences, and end with punctuation.

Alternatively, this doesn't seem to be used outside of the file in which it is declared, so it doesn't need to be exported at all.

Reply:
changed to `verifyRpmDB()` so it will not be exported.